### PR TITLE
Add /feed/lts and /feed/nightly

### PR DESCRIPTION
**Pull request by zudov**

### CODE REVIEW

**Comment by snoyberg** [comment]
How about making them /feed/lts and /feed/nightly instead?

**Comment by zudov** [comment]
@snoyberg Yeah, that would be better, will do now.

**Comment by zudov** [comment]
Would `/feed/#LtsMajor` (e.g. `/feed/lts-3) be useful?

**Comment by snoyberg** [comment]
Possibly someone would like it, sure

File: Stackage/Database.hs [hunk 1]
```diff
@@ -666,6 +668,37 @@ getSnapshots l o = run $ (,)
         []
         [LimitTo l, OffsetBy o, Desc SnapshotCreated]
 
+getLtsSnapshots :: GetStackageDatabase m
+                => Int -- ^ limit
+                -> Int -- ^ offset
+                -> m (Int, [Entity Snapshot])
+getLtsSnapshots l o = run $ do
+    ltsCount <- count ([] :: [Filter Lts])
+    snapshots <- E.select $ E.from $
```
Comment (zudov) [inline Stackage/Database.hs]: @snoyberg BTW, does that query looks good? I am not an SQL expert, perhaps there is a better way.

Comment (snoyberg) [inline Stackage/Database.hs]: LGTM

**Comment by zudov** [comment]
Is there something we can do to make travis cache work? The builds take way too much time.

**Comment by zudov** [comment]
Hmm, all the titles are the same at the moment, would fix it now.

**Comment by snoyberg** [comment]
I don't know why the builds aren't caching correctly. You can try pushing straight to master, which seems to cache better.

**Pull request merged**